src/test/java/com/google/api/generator/gapic/composer/resourcename/BUILD.bazel

```diff
@@ -0,0 +1,52 @@
+load("@rules_java//java:defs.bzl", "java_test")
```

Comment:
The `src/test/.../composer/resourcename` subfolder doesn't exist in `master`, should this be here?

Reply:
I moved resource related files into its own individual bazel package above. So I also move their tests into individual directory to match main function location.

Same for `default value composer`.

Reply:
`src/main/.../composer/resourcename`

Reply:
SG, could we please make the `resourcename` and `defalutvalue` changes as separate PRs in master? This will help keep the changes in `samplecode-master` restricted to only changes related to sample code features.

Note that we'll likely merge those in after 1/5, so we can release 0.0.15 first.

Reply:
Sure.

Reply:
See #630